app/views/students/status_updates/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Listing Status Updates
 
+= link_to 'New Status Update', [:new, :students, :status_update], class: 'btn btn-default'
+
+p
```

Comment:
It's terrible. But for now, the only way I am able to get some whitespace between the button and the table.